active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,181 @@
+In V1 the nodes have no option to open a drop-down menu, typically used to show the node's siblings.
+V2 will add this. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Represents a control that shows a list of items, typically the navigation trail to the current location.
+
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
+
+![Breadcrumb with nodes: Home, Documents, Design, Northwind, Images. The app is resized so that the Breadcrumb crumbles - then, clicking the ellipsis reveals the crumbled Home and Documents nodes](images/Breadcrumb_crumbling_flyout.gif)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Home", "Documents", "Design", "Northwind", "Images"}
+```
+
+If the last item in a Breadcrumb is text and is too long to fit it will be truncated. If not text this item will be clipped. 
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+## Breadcrumb.DropDownItemTemplate
+
+Gets or sets the template to display the drop-down of nodes
+
+The following configures a Breadcrumb so that it's drop down displays bold text.
+
+```xaml
+<Breadcrumb>
+    <Breadcrumb.DropDownItemTemplate>
+        <DataTemplate x:DataType='BreadcrumbDropDownItem`>
+            <TextBlock Text='{x:Bind Content} FontWeight="Bold" />
+        </DataTemplate>
+    </Breadcrumb.DropDownItemTemplate>
+</Breadcrumb>
+```
+
+## Other Breadcrumb members
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | Gets or sets the content of the Breadcrumb | null |
+| ItemTemplate | Gets or sets the template to display an item|  | 
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  | N/A |
+
+Spec note: the V2 of Breadcrumb will add flyouts from chevrons to view children of a node.
+Additional properties will be added to the API to enable this functionality. 
+
+# API Details
+
+```cs
+runtimeclass BreadcrumbItemClickedEventArgs
+{
+    Int32 Index { get; };
+    Object Item { get; };
+}
+
+unsealed runtimeclass BreadcrumbItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbItem();
+}
+
+unsealed runtimeclass BreadcrumbDropDownItem : Windows.UI.Xaml.Controls.ContentControl
+{
+    BreadcrumbDropDownItem();
+}
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass Breadcrumb : Windows.UI.Xaml.Controls.Control
+{
+    Breadcrumb();
+
+    Object DropDownItemTemplate{ get; set; };
+    Object ItemsSource{ get; set; };
+    Object ItemTemplate{ get; set; };
```

Comment:
What is this for?

Reply:
ItemTemplate ? how each item in the list of crumbs look like..

Reply:
Thanks. I added a diagram.